api-client/src/main/java/com/xing/api/data/profile/XingUser.java

```diff
@@ -699,7 +699,7 @@ public boolean isPremium() {
 
     /**
      * Returns {@code true} if the user is blacklisted, otherwise {@code false}. The assumption on which this method is
-     * based, is that XWS will always return a user id if the user not blocked or blacklisted.
+     * based, is that the API will always return a user id if the user not blocked or blacklisted.
```

Comment:
Since your here, please change `"user id if the user not blocked or blacklisted."` to `"<strong>user id</strong> if the user is not blocked or blacklisted."`